packages/react-native/src/primitives/Button/Button.tsx

```diff
@@ -36,16 +36,17 @@ export default function Button({
         disabled ? themedStyle.disabled : null,
         pressed ? themedStyle.pressed : null,
         pressedStateStyle,
-      ];
+      ] as StyleProp<ViewStyle>;
```

Comment:
Why do we need this cast now?

Reply:
B/c I touched this file so eslint gave me issues about implicit `any`s :/ Is there a better way to handle this?